frontend/src/screens/ProducScreen.js

```diff
@@ -19,67 +21,97 @@ const ProducScreen = ({ match }) => {
         dispatch(ProductDetails(match.params.id))
     }, [dispatch, match])
 
+    const addToCartHandler = () =>{
+        history.push(`/cart/${match.params.id}?pieces=${pieces}`)
+    }
+
     return (
         <>
             <Link className='btn btn-light my-3' to='/'>Back to</Link>
 
             {loading ? (<Loader />) : error ? (<Message variant='danger'>{error}</Message>) :
```

Comment:
avoid using nested ternary operator (use if statements to assign the element should be rendered to a variable before the return() and use the variable in the JSX)

Reply:
Okay i'm gonna edit it too